### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Novelnia**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/novelnia/background.png
Comment (DoJapHD) [inline minecraft_servers/novelnia/background.png:1]: Compress image

File: minecraft_servers/novelnia/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/novelnia/background@2x.png:1]: Compress image

File: minecraft_servers/novelnia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  "direct_ip": "novelnia.net",
+  "server_wildcards": [
+    "%.novelnia.net"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://novelnia.net",
+    "web_shop": "https://shop.novelnia.net",
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:13]: Cannot reach shop

File: minecraft_servers/novelnia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    "web_support": "https://support.novelnia.net",
+    "twitter": "Novelniachan",
+    "instagram": "novelnianet",
+    "youtube": "https://www.youtube.com/@novelnia",
+    "discord": "https://discord.gg/novelnia"
+  },
+  "gamemodes": {
+    "BedWars": {
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:21]: JSON conventions, key has to be lowercase

File: minecraft_servers/novelnia/manifest.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      "name": "BedWars",
+      "color": "#d51212"
+    },
+    "SkyWars": {
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:25]: JSON conventions, key has to be lowercase

File: minecraft_servers/novelnia/manifest.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      "name": "SkyWars",
+      "color": "#00e9fc"
+    },
+    "TTT": {
+      "name": "TTT",
+      "color": " #9f0101 "
+    },
+    "TryJump": {
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:33]: JSON conventions, key has to be lowercase

File: minecraft_servers/novelnia/manifest.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    },
+    "1vs1": {
+      "name": "1vs1",
+      "color": "#ffffff"
+    },
+    "KitPvP": {
+      "name": "KitPvP",
+      "color": "#00ffbd"
+    },
+    "Target": {
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:57]: JSON conventions, key has to be lowercase

File: minecraft_servers/novelnia/manifest.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      "name": "Target",
+      "color": "#ff6060"
+    },
+    "Community": {
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:61]: JSON conventions, key has to be lowercase

File: minecraft_servers/novelnia/manifest.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+      "color": "#c600a5"
+    }
+  },
+  "brand": {
+    "background": "#e04c51",
+    "text": "#ffffff"
+  },
+  "discord": {
+    "server_id": 851012884674641973,
+    "rename_to_minecraft_name": true
```
Comment (DoJapHD) [inline minecraft_servers/novelnia/manifest.json:72]: Reserved for partners, change to false

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by Novelnia** [comment]
I've made all the requested changes to the pull request. Could someone please review the updates?

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**